### ISSUE TITLE
Trained Lora error in diffusers with xform 

### ISSUE THREAD

**Issue by segatecm** [bmaltais/kohya_ss#1472]
With the new version of kohya to train lora.
diffusers 0.21.1, torch 2.0.1, xform 0.0.21, to gen images.

If the xform is open, the image is filled with random noise. if not used xform, the image is right.
But if i used the lora from others, it is all right.
Please help! 

here is the code:

def EvalLora(model_file, lora_file):
    pipe = DiffusionPipeline.from_pretrained(model_file, 
                                                torch_dtype=torch.float16,
                                                safety_checker = None,
                                                requires_safety_checker = False)
    pipe.load_lora_weights("lora", weight_name=lora_file)
    **pipe.enable_xformers_memory_efficient_attention()**
    pipe = pipe.to("cuda")
    gen = torch.Generator(device="cuda").manual_seed(123)
    prompt = "outdoor, A woman, black hair,18 years old,look at viewer, (best quality,High resolution,Masterpiece)"
    negative_prompt = "low res, low quality, jpg, ugly, multi person, nsfw, naked"
    image = pipe(prompt, 
                width = 768,
                height = 1024,
                generator = gen,
                    negative_prompt = negative_prompt).images[0]
    del pipe
    torch.cuda.empty_cache()
    image.save("buck.png")
if __name__ == "__main__":
    EvalLora("./diffuser-model/beautifulRealistic_brav5", "last-000005.safetensors")

train config:

{
  "LoRA_type": "Standard",
  "adaptive_noise_scale": 0,
  "additional_parameters": "",
  "block_alphas": "",
  "block_dims": "",
  "block_lr_zero_threshold": "",
  "bucket_no_upscale": true,
  "bucket_reso_steps": 64,
  "cache_latents": true,
  "cache_latents_to_disk": false,
  "caption_dropout_every_n_epochs": 0.0,
  "caption_dropout_rate": 0,
  "caption_extension": "",
  "clip_skip": "1",
  "color_aug": false,
  "conv_alpha": 1,
  "conv_block_alphas": "",
  "conv_block_dims": "",
  "conv_dim": 1,
  "decompose_both": false,
  "dim_from_weights": false,
  "down_lr_weight": "",
  "enable_bucket": true,
  "epoch": 20,
  "factor": -1,
  "flip_aug": false,
  "full_bf16": false,
  "full_fp16": false,
  "gradient_accumulation_steps": "1",
  "gradient_checkpointing": false,
  "keep_tokens": "0",
  "learning_rate": 0.0001,
  "logging_dir": "E:\\Images\\model1\\log",
  "lora_network_weights": "",
  "lr_scheduler": "cosine",
  "lr_scheduler_args": "",
  "lr_scheduler_num_cycles": "",
  "lr_scheduler_power": "",
  "lr_warmup": 10,
  "max_bucket_reso": 2048,
  "max_data_loader_n_workers": "0",
  "max_resolution": "512,512",
  "max_timestep": 1000,
  "max_token_length": "75",
  "max_train_epochs": "",
  "max_train_steps": "",
  "mem_eff_attn": false,
  "mid_lr_weight": "",
  "min_bucket_reso": 256,
  "min_snr_gamma": 0,
  "min_timestep": 0,
  "mixed_precision": "fp16",
  "model_list": "custom",
  "module_dropout": 0,
  "multires_noise_discount": 0,
  "multires_noise_iterations": 0,
  "network_alpha": 1,
  "network_dim": 64,
  "network_dropout": 0.1,
  "no_token_padding": false,
  "noise_offset": 0,
  "noise_offset_type": "Original",
  "num_cpu_threads_per_process": 2,
  "optimizer": "AdamW8bit",
  "optimizer_args": "",
  "output_dir": "E:\\Images\\model1\\model",
  "output_name": "last",
  "persistent_data_loader_workers": false,
  "pretrained_model_name_or_path": "D:\\stable-diffusion-webui\\models\\Stable-diffusion\\v1-5-pruned.ckpt",
  "prior_loss_weight": 1.0,
  "random_crop": false,
  "rank_dropout": 0,
  "reg_data_dir": "",
  "resume": "",
  "sample_every_n_epochs": 0,
  "sample_every_n_steps": 0,
  "sample_prompts": "",
  "sample_sampler": "euler_a",
  "save_every_n_epochs": 1,
  "save_every_n_steps": 0,
  "save_last_n_steps": 0,
  "save_last_n_steps_state": 0,
  "save_model_as": "safetensors",
  "save_precision": "fp16",
  "save_state": false,
  "scale_v_pred_loss_like_noise_pred": false,
  "scale_weight_norms": 0,
  "sdxl": false,
  "sdxl_cache_text_encoder_outputs": false,
  "sdxl_no_half_vae": true,
  "seed": "",
  "shuffle_caption": false,
  "stop_text_encoder_training": 0,
  "text_encoder_lr": 5e-05,
  "train_batch_size": 1,
  "train_data_dir": "E:\\Images\\model1\\img",
  "train_on_input": true,
  "training_comment": "",
  "unet_lr": 0.0001,
  "unit": 1,
  "up_lr_weight": "",
  "use_cp": false,
  "use_wandb": false,
  "v2": false,
  "v_parameterization": false,
  "v_pred_like_loss": 0,
  "vae_batch_size": 0,
  "wandb_api_key": "",
  "weighted_captions": false,
  "xformers": "xformers"
}
